poem_generator/generators/first_line/mbart_fi_first_line.py

```diff
@@ -0,0 +1,85 @@
+import logging
+from typing import List
+
+import torch
+from transformers import (
+    MBartTokenizer,
+    MBartForConditionalGeneration,
+)
+
+from poem_generator.io.candidates import PoemLine, PoemLineList
+from poem_generator.utils import remove_punct
+
+# This file contains the code for generating the next line with all supported models.
+# In future, it can become a package with several file (one for each implementation)
+
+BASE_MODEL = "facebook/mbart-large-cc25"
+MODEL_FILE = "models/first-line-fi-20-epochs/pytorch_model.bin"
+DEVICE = torch.device("cuda" if torch.cuda.is_available() else "cpu")
+
+
+def get_tokenizer():
+    tokenizer = MBartTokenizer.from_pretrained(
+        BASE_MODEL,
+        src_lang="fi_FI",
+        tgt_lang="fi_FI",
+    )
+
+    return tokenizer
+
+def get_model():
+    tokenizer = MBartTokenizer.from_pretrained(
+        BASE_MODEL,
+        src_lang="fi_FI",
+        tgt_lang="fi_FI",
+    )
+
+    logging.info("Loading base model {}".format(BASE_MODEL))
+    model = MBartForConditionalGeneration.from_pretrained(BASE_MODEL)
+    model.config.decoder_start_token_id = tokenizer.lang_code_to_id["fi_FI"]
+
+    model.resize_token_embeddings(len(tokenizer))  # is this really necessary here?
+    logging.info("Model vocab size is {}".format(model.config.vocab_size))
+    model.load_state_dict(torch.load(MODEL_FILE, map_location=torch.device(DEVICE)))
+    model.to(DEVICE)
+
+    return model
+
+
+def filter_candidates(candidates: List[str]):
```

Comment:
This is duplicate, can you move it to `poem_generator/utils.py` and remove it also from `poem_generator/generators/next_line/mbart_fi_single_line.py`?